database_schema.md

```diff
@@ -137,8 +137,34 @@ seat_id: {
 ```
 
 ### Matter
-A matter is specifically a legislative matter. A bill, resolution, initiative, etc. It has a sponser which may be a
-person or a body.
+A matter is specifically a legislative matter. A bill, resolution, initiative, etc.
+```
+matter_id: {
+    name: str
+    matter_type_id: str
+    title: str
+    status: str
+    agenda_date: datetime
```

Comment:
Can we change this to `most_recent_event`
```suggestion
    most_recent_event: {
        id: str
        body: str
        datetime: datetime
    }
```

Reply:
And potentially add an entirely new key for `upcoming_event`?

Reply:
Sure, as long we would be able to filter by the date.
.where("matter.most_recent_event.datetime", "<", now) would be possible on firestore?

Reply:
Good point. Will have to check.

Reply:
Looks like we can?
https://fireship.io/lessons/advanced-firestore-nosql-data-structure-examples/
"db.collection('tweets').where('tags.angular', '==', true);"